package.json

```diff
@@ -75,9 +74,9 @@
     "react-dom": "^16.6.1",
     "react-load-script": "^0.0.6",
     "react-redux": "^5.0.7",
-    "react-router": "^4.4.0-beta.6",
```

Comment:
That's not actually the latest version of react-router. I didn't want to go to the trouble of rewriting all the routing API at this stage. Not sure why we a referencing the beta version here. Looks like there never was a stable version of react router 4.4 tagged.